docs/content/build-image.md

```diff
@@ -20,10 +20,10 @@ To create a new CNAB with Porter, you first run `porter create`. The generated `
 name: HELLO
 version: 0.1.0
 description: "An example Porter configuration"
-# TODO: update the registry to your own, e.g. myregistry/porter-hello:v0.1.0
-tag: getporter/porter-hello:v0.1.0
+# TODO: update the registry to your own, e.g. myregistry/porter-hello
+tag: getporter/porter-hello
 
-# Uncomment the line below to use a template Dockerfile for your invocation image
+# Uncomment the line below to customize the Dockerfile, see https://porter.sh/custom-dockerfile/
```

Comment:
Would recommend not including the comments in the build-image.md documentation to limit what needs to get updated when changes to the base template are made.